cli/deploy-moe-minimal-single-model-registered.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+#set -e
```

Comment:
Could you uncomment this? This will bypass our build test.
```suggestion
set -e
```